packages/x-data-grid-premium/src/tests/aggregation.DataGridPremium.test.tsx

```diff
@@ -641,7 +641,7 @@ describe('<DataGridPremium /> - Aggregation', () => {
     it('should use the aggregation function valueFormatter if defined', () => {
       const customAggregationFunction: GridAggregationFunction = {
         apply: () => 'Agg value',
-        valueFormatter: (params) => `+ ${params.value}`,
+        valueFormatter: (value) => `+ ${value}`,
```

Comment:
Probably document this in the migration guide and release notes

Reply:
It's already in. This was missed during the refactor.